kong/plugins/opentelemetry/handler.lua

```diff
@@ -164,31 +145,48 @@ function OpenTelemetryHandler:header_filter(conf)
   end
 end
 
+local function get_queue_params(config)
+  local queue_config = unpack({ config.queue or {}})
```

Comment:
How is this different from:
```suggestion
  local queue_config = config.queue or {}
```